src/cli/CommandPull.hpp

```diff
@@ -110,6 +111,20 @@ class CommandPull : public Command {
         cli::utility::printLog(boost::format("successfully parsed CLI arguments"), common::LogLevel::DEBUG);
     }
 
+    bool serverIsInsecure(std::string server) {
```

Comment:
I think the name of this function is somewhat misleading. `serverIsInsecure` might suggest that the function performs some kind of analysis to determine whether the server is secure or not (in a general sense), while in reality the function checks if the server in input corresponds to one of the servers which are permitted to be insecure from the configuration.

I think that a name like `serverIsAllowedInsecure` or something similar would be more accurate.
What do you think?

Reply:
I agree, changed.